cfme/tests/containers/test_configurable_menus.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.containers.provider import ContainersProvider
+from cfme import BaseLoggedInPage
+from utils import testgen
+from utils.version import current_version
+from utils.appliance.implementations.ui import navigate_to
+from utils.browser import ensure_browser_open
+from utils.blockers import BZ
+
+
+pytestmark = [pytest.mark.uncollectif(lambda: current_version() < "5.8")]
+pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
+
+
+def config_menus_visibility(appliance, is_visible):
+    # Configure the menus visibility parameters in config/settings.yaml
+    yaml_config = appliance.get_yaml_config()
+
+    if (yaml_config['product']['datawarehouse_manager'] == is_visible) and \
+            (yaml_config['prototype']['monitoring'] == is_visible):
+        return  # If this is already the state, return
+
+    yaml_config['product']['datawarehouse_manager'] = is_visible
+    yaml_config['prototype']['monitoring'] = is_visible
+
+    appliance.set_yaml_config(yaml_config)
+    appliance.reboot()
+    ensure_browser_open()
```

Comment:
You probably want to do the browser quit here. @RonnyPfannschmidt what is the approach with the browser manager here?